pygeo/parameterization/DVGeo.py

```diff
@@ -1685,6 +1686,15 @@ def update(self, ptSetName, childDelta=True, config=None):
         # We've postponed things as long as we can...do the finalization.
         self._finalize()
 
+        for iChild in range(len(self.children)):
+            if len(self.children[iChild].axis) > 0:
+                self.children[iChild]._finalize()
```

Comment:
As you mentioned, the `_finalize` here is needed to get the `refAxis.coef` (coefficient of refAxis nodes) that are then embedded in the parent control volume below.
I am wondering then:
1. if it still makes sense to have another `child._finalize()` [here](https://github.com/mdolab/pygeo/blob/23150ad5f348f1574612e68ea43ff68277424173/pygeo/parameterization/DVGeo.py#L1780)
2. what caused the mismatch in the previous tests - was it due to the fact that the control points themselves were not embedded or something? I am honestly a bit confused by how pyGeo currently works, maybe there are some other intermediate steps we are missing if the axis is not added rightaway

Reply:
1. The finalize method exists right away if it has already been called. This it is safe to repeatedly call finalize during each call of the update method.

Reply:
2. I don't think I know which tests you are referring to.

Reply:
I was referring to the test failures after the first commit, but the build log is lost now.
Agreed on the multiple finalizing calls.